server/game/cardImplementations/01_SOR/ViperProbeDroid.ts

```diff
@@ -0,0 +1,23 @@
+import AbilityHelper from '../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../core/card/NonLeaderUnitCard';
+
+export default class ViperProbeDroid extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '8986035098',
+            internalName: 'viper-probe-droid'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'Look at an opponent\'s hand.',
+            immediateEffect: AbilityHelper.immediateEffects.lookAt((context) => ({
+                target: context.player.opponent.hand.sort((a, b) => a.name.localeCompare(b.name)),
```

Comment:
Any specific reason we want them sorted?

Reply:
@AMMayberry1 It was like that in the L5R code. I don't think it's necessary, but I suspect the reason is so that the cards are not in draw order. This would give someone information about how long someone has held a card etc.